### PR TITLE
Adding doco/snippets for BrokeredMessageBodyConventionUsage

### PR DESCRIPTION
**Pull request by SeanFeldman**
@Particular/azure please review

### CODE REVIEW

File: nservicebus/azure/azure-servicebus-transport.md [hunk 1]
```diff
@@ -81,6 +81,17 @@ NOTE: `QueueName` and `QueuePerInstance` are obsoleted. Instead, use bus configu
 
 Defaults are just starting values. You should always measure and test these values against your solution and adjust those accordingly.
 
+### BrokeredMessage body conventions
+
+By default BrokeredMessage body is transmitted as a byte array. For scenarios such as native integration, the body can be stored and retrieved using an alternative - `Stream`. To specify how the `BrokeredMessage` body is stored an retrieved, override conventions provided by `BrokeredMessageBodyConversion` class.
```
Comment (SimonCropp) [inline nservicebus/azure/azure-servicebus-transport.md]: > by `BrokeredMessageBodyConversion` class.

perhaps

> by using the `BrokeredMessageBodyConversion` class.

**Comment by SimonCropp** [comment]
i think you forgot to push the csproj

File: nservicebus/azure/azure-servicebus-transport.md (hunk 1 above)
Comment (SimonCropp) [inline nservicebus/azure/azure-servicebus-transport.md]: so one "BrokeredMessage" is code bracketed but not the other one?

**Comment by SeanFeldman** [comment]
Thank you @SimonCropp, indeed

File: nservicebus/azure/azure-servicebus-transport.md (hunk 1 above)
Comment (SeanFeldman) [inline nservicebus/azure/azure-servicebus-transport.md]: Fixed

**Comment by yvesgoeleven** [comment]
@SeanFeldman looks good

**Comment by SimonCropp** [comment]
@SeanFeldman same here. pull the trigger

**Comment by SeanFeldman** [comment]
Thank you gentlemen

**Pull request merged**